source/dem/pp_nonlinear_force.cc

```diff
@@ -111,9 +111,17 @@ PPNonLinearForce<dim>::calculate_pp_contact_force(
           Tensor<1, dim> normal_force;
           normal_force = spring_normal_force + dashpot_normal_force;
 
-          double maximum_tangential_overlap =
-            physical_properties.friction_coefficient_particle *
-            normal_force.norm() / tangential_spring_constant;
+          double maximum_tangential_overlap;
+          if (tangential_spring_constant != 0.0)
```

Comment:
Are you sure that tangential_spring_constant is always positive? This variable is the stiffness of the spring?

Reply:
Thanks for reminding the particle IDs. After I recloned the code, I forgot to recorrect the IDs in two tests. I corrected it and repushed it.
It is the tangential stiffness of the spring. It should be positive, since it is defined as (for non-linear model for instance):
kt = 8 * shear_modulus * sqrt(r * normal_overlap)